src/tests/ftest/osa/osa_online_parallel_test.py

```diff
@@ -37,6 +37,7 @@
 from command_utils import CommandFailure
 from mpio_utils import MpioUtils
 from daos_racer_utils import DaosRacerCommand
+from apricot import skipForTicket
```

Comment:
(pylint-reimported) Reimport 'skipForTicket' (imported line 32)
(pylint-ungrouped-imports) Imports from package apricot are not grouped